components/rhel/misc/rhel-k8s-singlenode-setup/README.rst

```diff
@@ -0,0 +1,31 @@
+Vagrantfile: Single Node Kubernetes
+===================================
+
+This Vagrantfile is the suggested configuration for using the CDK with Kubernetes.  This file sets up private networking that will be used to expose the docker daemon and kubernetes to the host.  This file also sets up a single node Kubernetes instance where the master and node are on the same machine.
+
+This Vagrantfile is useful for anyone using host-based tools, such as the `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_ or the kubernetes and docker CLIs, with the CDK.
+
+QuickStart
+----------
+
+1. Get latest CDK box and add it to vagrant.
```

Comment:
@navidshaikh nope, this is not required. Lets not try to write documentation for CDK. In future we can just point to the documentation when it is available.